gadfly/sentence_summarizer.py

```diff
@@ -58,3 +59,92 @@ def summarize(self, sents, n):
     def _rank(self, ranking, n):
         """ return the first n sentences with highest ranking """
         return nlargest(n, ranking, key=ranking.get)
+
+
+class TF_IDFSummarizer:
+    def __init__(self):
+        pass
+
+    def get_tokens_and_freqs(self, sents, style):
+        """Get a list of all tokens in sents and a Counter of them"""
+        if style == 'standard':
+            all_tokens = [token.text for sent in sents for token in sent]
+        elif style == 'lower':
+            all_tokens = [token.lower_ for sent in sents for token in sent]
+        elif style == 'lemma':
+            all_tokens = [token.lemma_ for sent in sents for token in sent]
+        else:
+            print("Please enter an appropriate option for 'style'.")
+        document_freq_dict = Counter(all_tokens)
+        return document_freq_dict, all_tokens
+
+    def get_dict_tf_idf(self, sents, document_freq_dict, all_tokens, style):

```

Comment:
Shouldn't this method be agnostic of the style parameter? It shouldn't need all the if else checks you have below.